wsd/Admin.cpp

```diff
@@ -294,6 +295,10 @@ void AdminSocketHandler::handleMessage(const std::vector<char> &payload)
         // Let's send back the current log levels in return. So the user can be sure of the values.
         sendTextFrame("channel_list " + _admin->getChannelLogLevels());
     }
+    else if (tokens.equals(0, "clusterresize"))
+    {
+        COOLWSD::alertAllUsersInternal("updateroutetoken");
```

Comment:
So - looks like this might work -but- it will create a huge volume of requests - eg. 100k of them for a big server.
In almost all of these cases - we want to do the same thing for all of these users: to update all the client's routing token to a different string.

Can we not let this server know what its new routing token/URL is -and- then just tell all the clients what it is - without needing to migrate anything and/or send a server request to ask for a new routing token ?

ie. we just update our internal state / URL in the JS client without needing to ask the server again - and without re-connecting, and/or doing anything expensive or disruptive to the websocket - we shouldn't even need a refresh.

Thanks

Reply:
Thanks, I have updated the PR now COOL recieves a JSON with mapping of new serverId and routeToken , then based on that map COOL can update the routeToken/URL